src/scores/categorical/contingency_impl.py

```diff
@@ -33,14 +35,25 @@
 
 class BasicContingencyManager:  # pylint: disable=too-many-public-methods
     """
-    A BasicContingencyManager is produced when a BinaryContingencyManager is transformed.
+    See https://scores.readthedocs.io/en/stable/tutorials/Binary_Contingency_Scores.html for
```

Comment:
Given we have been moving to relative links where possible, can/should this be a relative path?

Reply:
I don't think relative pathing is possible from API docs, but I would be happy to be corrected (and shown how!)

Reply:
@nikeethr seems like a good detective on these questions... :) If you're not sure, that's fine too :)

Reply:
Have you tried `:doc:`? https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role